Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -178,6 +178,7 @@ define([
         this._minimumLevel = defaultValue(options.minimumLevel, 0);
         this._maximumLevel = options.maximumLevel;
         this._minimumRetrievingLevel = defaultValue(options.minimumRetrievingLevel, 0);
+        this._maximumRetrievingLevel = options.maximumRetrievingLevel;
```

Comment:
You can use  `= defaultValue(options.maximumRetrievingLevel, Infinity);` and remove `defined(...` in `requestImage`